entrypoint.sh

```diff
@@ -14,6 +14,8 @@ chmod 600 $TEMP_SSH_PRIVATE_KEY_FILE
 
 echo 'sftp start'
 
+# create directory if needed
+ssh -i $TEMP_SSH_PRIVATE_KEY_FILE $1@$2 mkdir -p $5
```

Comment:
@JulianGro $5 should be local path, why you are using $5 instead of $6?

Reply:
Yeah there is an issue here.
In my case I do not define the port and it uses the standard value, but apparently it appends the port to the end of the variables when you don't define it. So in my case remote path *is* actually $5 even though it should be $6